lib/quicly.c

```diff
@@ -4691,7 +4691,7 @@ static int handle_ack_frame(quicly_conn_t *conn, struct st_quicly_handle_payload
         /* Ack blocks are organized in the ACK frame and consequently in the ack_block_lengths array from the largest acked down.
          * Processing acks in packet number order requires processing the ack blocks in reverse order. */
         uint64_t pn_block_max = pn_acked + frame.ack_block_lengths[gap_index] - 1;
-        QUICLY_PROBE(QUICTRACE_RECV_ACK, conn, conn->stash.now, pn_acked, pn_block_max);
+        QUICLY_PROBE(ACK_BLOCK_RECEIVED, conn, conn->stash.now, pn_acked, pn_block_max);
```

Comment:
```suggestion
        QUICLY_PROBE(ACK_BLOCK_RECEIVE, conn, conn->stash.now, pn_acked, pn_block_max);
```

just to be consistent with other RECEIVE events

Reply:
So the use of past tense here was intentional. A week or so ago @hfujita raised a question about using past tense, which @kazuho and I agreed because the use of past tense would follow QLog's naming convention. 84d322d is an example result of this.

Reply:
I agree that it is better. We should make this consistent though, since we now have _SEND, _SENT, _RECEIVE as various probe names.